src/actions/formActions.js

```diff
@@ -0,0 +1,55 @@
+import routesPaths from 'constants/routesPaths';
+import api from 'api';
+import swallow from 'utils/swallow';
+import { CALL_API, Schemas } from '../middleware/api';
+
+export const FORMS_REQUEST = 'FORMS_REQUEST';
+export const FORMS_SUCCESS = 'FORMS_SUCCESS';
+export const FORMS_FAILURE = 'FORMS_FAILURE';
+
+const fetchForms = (category, refetch) => ({
+  category,
+  [CALL_API]: {
+    types: [FORMS_REQUEST, FORMS_SUCCESS, FORMS_FAILURE],
+    endpoint: `form_categories/${category}/forms`,
```

Comment:
should the endpoint's url be defined in here?

Reply:
I don't see it necessary to extract it yet because currently, only the action needs to know which endpoint to use